src/openziti/zitisock.py

```diff
@@ -20,14 +20,40 @@
 from . import context, zitilib
 
 
+def process_bindings(orig):
```

Comment:
how hard would it be to get some tests added for this bit of code? it'd be helpful to see what sort of inputs you tried and it'd be just 'a good idea' overall, I'm sure you'd agree. :)